dpsim-models/src/Base/Base_ReducedOrderSynchronGenerator.cpp

```diff
@@ -273,6 +281,25 @@ void Base::ReducedOrderSynchronGenerator<VarType>::setInitialValues(
 	mInitCurrent = mInitCurrent / mBase_I_RMS;
 
 	mInitialValuesSet = true;
+
+	SPDLOG_LOGGER_INFO(this->mSLog,
```

Comment:
Soon, Initialization values are going to be available only under `DEBUG`. Please this change already here.

Reply:
Why did you leave it on `INFO`?